go/kbfs/simplefs/simplefs.go

```diff
@@ -2376,7 +2376,19 @@ func (k *SimpleFS) SimpleFSSyncConfigAndStatus(
 			res.Folders[j].Folder.ToString()
 	})
 
-	// TODO(KBFS-4067): fill in the active overall prefetch progress.
+	if len(tlfIDs) > 0 {
+		p := k.config.BlockOps().Prefetcher().OverallSyncStatus()
+		res.OverallStatus.PrefetchProgress =
+			k.prefetchProgressFromByteStatus(p)
+		if p.SubtreeBytesTotal == p.SubtreeBytesFetched ||
+			p.SubtreeBytesTotal == 0 {
+			res.OverallStatus.PrefetchStatus = keybase1.PrefetchStatus_COMPLETE
+		} else {
+			res.OverallStatus.PrefetchStatus =
+				keybase1.PrefetchStatus_IN_PROGRESS
+		}
```

Comment:
Hmm do we never have `keybase1.PrefetchStatus_NOT_STARTED`?

Reply:
I guess if all of the individual folders are NOT_STARTED, then the overall one should be not started.  I'll fix it up.

Reply:
(But as it is, if there are no TLFs configured, it will be NOT_STARTED since that's the default.)